connectivity/drivers/ble/FEATURE_BLE/TARGET_STM32WB/HCIDriver.cpp

```diff
@@ -480,10 +483,29 @@ class TransportDriver : public CordioHCITransportDriver {
             WirelessFwInfo_t wireless_info_instance;
             WirelessFwInfo_t *p_wireless_info = &wireless_info_instance;
             if (SHCI_GetWirelessFwInfo(p_wireless_info) != SHCI_Success) {
-                tr_info("SHCI_GetWirelessFwInfo error");
+                tr_error("SHCI_GetWirelessFwInfo error");
             } else {
+                // https://github.com/STMicroelectronics/STM32CubeWB/tree/master/Projects/STM32WB_Copro_Wireless_Binaries
+                // Be sure that you are using the latest BLE FW version
                 tr_info("WIRELESS COPROCESSOR FW VERSION ID = %d.%d.%d", p_wireless_info->VersionMajor, p_wireless_info->VersionMinor, p_wireless_info->VersionSub);
-                tr_info("WIRELESS COPROCESSOR FW STACK TYPE = %d", p_wireless_info->StackType);
+                tr_info("WIRELESS COPROCESSOR FW STACK TYPE = %d (ROM size 0x%x)", p_wireless_info->StackType, MBED_ROM_SIZE);
+
+#if STM32WB55xx
```

Comment:
why is this part of code limited to WB55xx (not applicable at other derivatives ? )

Reply:
yes, for coming STM32WB15 for ex, only HCI  FW will be supported, and FW size is different